src/cobra/test/test_core/test_core_reaction.py

```diff
@@ -54,7 +54,10 @@ def test_gpr_uppercase():
 
 @pytest.mark.parametrize("input_gpr", ["(a1 or a2", "(forT or "])
 def test_gpr_malformed(input_gpr):
```

Comment:
Please add the argument type annotation as `str` and return type annotation as `None`.